src/screens/CreatePost/index.js

```diff
@@ -187,9 +188,6 @@ const CreatePost = () => {
 
   React.useEffect(() => {
     init();
-    if (isInCreatePostTopicScreen) {
-      setTimeout(() => onChangeText(`${message}#${listTopic[0]}`), 500);
-    }
   }, []);
 
   const init = async () => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the condition inside the `useEffect` hook simplifies the logic and makes it easier to understand. However, make sure that the removed condition is not needed anymore or has been handled elsewhere.
```diff
-    if (isInCreatePostTopicScreen) {
-      setTimeout(() => onChangeText(`${message}#${listTopic[0]}`), 500);
-    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->